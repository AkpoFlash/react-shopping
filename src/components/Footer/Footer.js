import React from 'react';
import styled from 'styled-components';

const Footer = (props) => {

	const Footer = styled.footer`
		background-color: var(--secendaryColor);
		grid-row: 4;
		grid-column: 2;
		width: 100%;
		text-align: center;
		line-height: 50px;
	`;

	return (
		<Footer>
			Copyright &#169; {(new Date()).getFullYear()}
		</Footer>
	);
}

export default Footer;