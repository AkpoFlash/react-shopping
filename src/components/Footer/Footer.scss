.footer{
	background-color: var(--secendaryColor);
	grid-row: 4;
	grid-column: 2;
	width: 100%;
	text-align: center;
	line-height: 50px;
}