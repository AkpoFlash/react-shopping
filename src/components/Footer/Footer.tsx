import * as React from 'react';
import styled from 'styled-components';

import { COLOR_GRAY } from '../../constants/styles';

const StyledFooter = styled.footer`
	background-color: ${ COLOR_GRAY };
	grid-row: 3;
	grid-column: 2;
	width: 100%;
	text-align: center;
	line-height: 50px;
`;

export const Footer: React.FunctionComponent<any> = (props) => {
	return (
		<StyledFooter>
			Copyright &#169; {(new Date()).getFullYear()}
		</StyledFooter>
	);
};

export default React.memo(Footer);