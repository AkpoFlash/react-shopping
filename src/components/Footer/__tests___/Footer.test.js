import React from 'react';
import { render } from 'enzyme';

import Footer from '../Footer';

describe('Footer', () => {
	const footer = render(<Footer />);

	it('should Footer not changed', () => {
		expect(footer).toMatchSnapshot();
	});
});