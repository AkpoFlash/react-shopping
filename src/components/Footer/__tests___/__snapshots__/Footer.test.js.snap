// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should Footer not changed 1`] = `
<footer
  class="sc-bdVaJa bUietH"
>
  Copyright Â© 2019
</footer>
`;
